src/Compilers/Core/Portable/Symbols/ITypeSymbol.cs

```diff
@@ -95,6 +96,21 @@ public interface ITypeSymbol : INamespaceOrTypeSymbol
         /// Must be a non-null interface property, method, or event.
         /// </param>
         ISymbol FindImplementationForInterfaceMember(ISymbol interfaceMember);
+
+        /// <summary>
+        /// True if the type is ref-like, meaning it follows rules similar to CLR by-ref variables. False if the type
+        /// is not ref-like or if the language has no concept of ref-like types.
```

Comment:
>the language has no concept of ref-like types [](start = 34, length = 45)

We only have two languages.  I believe both of them are capable of returning `true` for this sometimes.  If that's right, then this part of the sentence is unhelpful.